app/grandchallenge/algorithms/templates/algorithms/result_list.html

```diff
@@ -54,97 +54,6 @@ <h2>Results for {{ algorithm.title }}</h2>
-                                              title="View algorithm result">
-                                            <i class="fa fa-eye"></i> View Result
-                                        </span>
-                                    </a>
-                                </li>
-                            {% endif %}
-                            {% for image in result.images.all %}
-                                {% for file in image.files.all %}
-                                    <li>
-                                        <a href="{{ file.file.url }}">
-                                            <span class="badge badge-primary" title="Download Output {{ forloop.parentloop.counter }} {{ image.name|stem }} ({{ file.file|suffix }})">
-                                                <i class="fa fa-download"></i> Output {{ forloop.parentloop.counter }} ({{ file.file|suffix }})
-                                            </span>
-                                        </a>
-                                    </li>
-                                {% endfor %}
-                            {% empty %}
-                                <li>
-                                    <span class="badge badge-warning"
-                                          title="No output images were found">
-                                        <i class="fa fa-eye-slash"></i>
-                                    </span>
-                                </li>
-                            {% endfor %}
-                            {% for file in result.job.image.files.all %}
-                                <li>
-                                    <a href="{{ file.file.url }}">
-                                        <span class="badge badge-primary" title="Download Input {{ result.job.image.name|stem }} ({{ file.file|suffix }})">
-                                            <i class="fa fa-download"></i> Input ({{ file.file|suffix }})
-                                        </span>
-                                    </a>
-                                </li>
-                            {% endfor %}
-                        </ul>
-                    </td>
-                    <td>
-                        {% if result.comment %}
-                            <a href="#resultInfoModal"
-                               class="badge badge-primary"
-                               data-toggle="modal"
-                               data-target="#resultInfoModal"
-                               data-title="Result Comment"
-                               data-output="{{ result.comment }}"
-                               data-pk="{{ result.pk }}"
-                               title="Result Comment">
-                                <i class="fa fa-file"></i>
-                            </a>
-                        {% endif %}
-                        {% if change_result|get_jsonpath:result.pk %}
```

Comment:
`|get_jsonpath:result.pk` has been lost from the refactoring, are you sure that the permissions check is correct? It could be that a user only has permission to change one of the results.

Reply:
The check is now added on a row by row basis. The rows are formatted one by one, so `change_result` now only contains the value for this particular entry